examples/ccm-demo/README.md

```diff
@@ -1,4 +1,8 @@
-# AES-CCM demo
+# The cipher block chaining - message authentication code (CCM) mode demo - Encryption
```

Comment:
These titles seem too verbose

```suggestion
# CCM peripheral encryption demo
```

The acronym can be expanded in the text below

(same goes for the other readmes)